bundle/bundle.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2024 Canonical Ltd.
```

Comment:
not for this PR.. but sooner rather than later we will have to provide a terraform spec..